packages/browser-crawler/package.json

```diff
@@ -58,6 +58,7 @@
         "@crawlee/browser-pool": "^3.3.2",
         "@crawlee/types": "^3.3.2",
         "@crawlee/utils": "^3.3.2",
+        "idcac-playwright": "^0.1.1",
```

Comment:
this will need to go to the `playwright-crawler` and `puppeteer-crawler` packages instead, as that is where you are importing it